eth/vm/forks/frontier/state.py

```diff
@@ -128,14 +136,16 @@ def build_computation(self,
                     encode_hex(message.storage_address),
                 )
             else:
+                with self.vm_state.trace(tracer):
+                    computation = self.vm_state.get_computation(
+                        message,
+                        transaction_context,
+                    ).apply_create_message()
```

Comment:
An alternative might be to pass the tracer into `get_computation` or `apply_create_message`